tests/core/math/test_vector3.h

```diff
@@ -195,6 +206,14 @@ TEST_CASE("[Vector3] Normalization methods") {
 	CHECK_MESSAGE(
 			Vector3(1, 1, 1).normalized().is_equal_approx(Vector3(Math_SQRT13, Math_SQRT13, Math_SQRT13)),
 			"Vector3 normalized should work as expected.");
+	Vector3 vector = Vector3(3.2, -5.4, 6);
+	vector.normalize();
+	CHECK_MESSAGE(
+			vector == Vector3(3.2, -5.4, 6).normalized(),
+			"Vector2 normalize should convert same way as Vector2 normalized.");
+	CHECK_MESSAGE(
+			vector.is_equal_approx(Vector3(0.368523, -0.621882, 0.69098)),
+			"Vector2 normalize should work as expected.");
```

Comment:
1. You accidentally wrote Vector2 instead of Vector3 here.
2. We should use high precision for our test values, so that the test will still pass if we ran the test with a higher precision and a stricter tolerance. I usually just use [this website](https://keisan.casio.com/calculator), manually enter the formulas, set the precision to like a hundred, and then copy the first 20 or so characters. As a sanity check you can check that it's approximately the same as the values you've already calculated here.

Reply:
I added constructor tests for Vector4 and Vector4i, and as for the more precise numbers, I have put a few, but not all, because I may go insane.